distribution/docs/src/main/jdocs/content/_security-framework/realms-contents.adoc

```diff
@@ -0,0 +1,107 @@
+:title: Realms
+:type: subSecurityFramework
+:status: published
+:parent: Subject
+:order: 01
+:summary: Shiro realms.
+
+${branding} uses Realms for Authentication and Authorization.
```

Comment:
Should say that we use "Shiro Realms" here. We also have JAAS realms and the realms on the web context policy which are different from these realms.

Reply:
updated